test/crosschain/EthTxAuth.test.ts

```diff
@@ -138,14 +132,12 @@ describe('L1 interaction with Snapshot X', function () {
     await starknet.devnet.loadL1MessagingContract(networkUrl, mockStarknetMessaging.address);
     await starknetCommit
       .connect(signer)
-      .commit(
-        getCommit(BigInt(space.address), BigInt(getSelectorFromName('vote')), proposeCalldata)
-      ); // Wrong selector
+      .commit(getCommit(BigInt(space.address), PROPOSE_SELECTOR, proposeCalldata)); // Wrong selector
```

Comment:
This was supposed to be `VOTE_SELECTOR`, not `PROPOSE_SELECTOR` I think (since the comment says "Wrong selector") ?

Reply:
The test was testing that revert case - the comment is just referring to what I changed to cause it to fail.